front/lib/validators/default.js

```diff
@@ -0,0 +1,7 @@
+import { number, string } from "yup"
+
+export const stringValidator = string()
+export const descriptionValidator = string().min(20)
+export const idValidator = number()
+export const numberValidator = number()
```

Comment:
Same as above

Reply:
I'll be removing this, it's also not used in fact. Thanks